models/palette_model.py

```diff
@@ -392,6 +390,7 @@ def set_input(self, data):
                         self.device,
                         batched=True,
                     )
+                    self.y_t = fill_mask_with_random(self.gt_image, self.mask, -1)
```

Comment:
Let's add a comment that the sam related stuff shall go into the dataloader, eventually.